src/server/api/routes/users.router.js

```diff
@@ -63,4 +63,45 @@ router.get('/:id', (req, res, next) => {
     .catch(next);
 });
 
+/**
+ * @swagger
+ * /users/{ID}:
+ *  patch:
+ *    tags:
+ *    - Delivery Update
```

Comment:
should be  Users

Reply:
I wrote the delivery update on purpose. Because the information entered changes only delivery address